spec/hardware/cpu.fmf

```diff
@@ -115,15 +118,20 @@ example:
     cpu:
         hyper-threading: true
 
+  - |
+    # Request a CPU with specified frequency.
```

Comment:
There is already the frequency example above, stating MHz is default and that it accepts different units, maybe best to not add another example?
https://tmt.readthedocs.io/en/stable/spec/hardware.html#cpu

Reply:
Updated,thanks for your review:)